src/projects/GroceriStar/groceristar.js

```diff
@@ -33,9 +33,9 @@ const getUsers = function () {
   // return __get(users)
 }
 
-// const getDepartments = function () {
-//   return __get(departments)
-// }
+const getDepartments = () => {
+  return __get(departments)
```

Comment:
if we import departments at the beginning - we actually don't need these `__get` and `getXXX()` methods now